src/group_theory/subgroup.lean

```diff
@@ -299,6 +299,10 @@ def subtype : H →* G := ⟨coe, rfl, λ _ _, rfl⟩
 
 @[simp, to_additive] theorem coe_subtype : ⇑H.subtype = coe := rfl
 
+@[to_additive] lemma subtype_injective {H : subgroup G} :
+  function.injective H.subtype :=
+by { unfold function.injective, intros a b, simp }
+
```

Comment:
```suggestion
@[to_additive] lemma subtype_injective {H : subgroup G} :
  function.injective H.subtype :=
subtype.coe_injective

```
although it's not clear that you would ever need this lemma instead of just using `subtype.coe_injective` directly.

Reply:
I must have missed `coe_injective`